ms/MSOper/MSMetaData.cc

```diff
@@ -3878,17 +3878,23 @@ std::pair<MDirection, MDirection> MSMetaData::getPointingDirection(
 		row >= this->nRows(),
 		"Row number exceeds number of rows in the MS"
 	);
-	SHARED_PTR<Vector<Int> > ant1, ant2;
-	_getAntennas(ant1, ant2);
-	antenna1 = (*ant1)[row];
-	antenna2 = (*ant2)[row];
-	time = (*_getTimes())[row];
+	// KS note: 2015/10/30
+	// getColum method (also used in _getAntennas and _getTimes) is
+	// too expesive for the current purpose of getting single cell.
+	String ant1ColName = MeasurementSet::columnName(MSMainEnums::ANTENNA1);
+	String ant2ColName = MeasurementSet::columnName(MSMainEnums::ANTENNA2);
```

Comment:
Julian, I believe what you mean by this is, something like below. Is it correct?
I see there are several other calls to MeasurementSet::columnName  in other functions of MSMetaData class, which do not use const reference.
Do you think I'd better modify those too?

```javascript
-       String ant1ColName = MeasurementSet::columnName(MSMainEnums::ANTENNA1);
-       String ant2ColName = MeasurementSet::columnName(MSMainEnums::ANTENNA2);
+       const String& ant1ColName = MeasurementSet::columnName(MSMainEnums::ANTENNA1);
+       const String& ant2ColName = MeasurementSet::columnName(MSMainEnums::ANTENNA2);
        antenna1 = ROScalarColumn<Int>(*_ms, ant1ColName).get(row);
        antenna2 = ROScalarColumn<Int>(*_ms, ant2ColName).get(row);
```